vizro-core/src/vizro/static/css/variables.css

```diff
@@ -27,6 +27,8 @@
   --text-weight-light: 300;
   --text-weight-regular: 400;
   --text-weight-semibold: 600;
+  --root-loading-color: rgba(20, 23, 33, 1);
+  --root-loading-contrast: rgba(255, 255, 255, 1);
```

Comment:
```suggestion
```

I think no need to store in variables as we don't re-use this. I would just hard-code into the css file